src/services/user/model/user.schema.js

```diff
@@ -102,6 +103,7 @@ const userSchema = new Schema(
 );
 
 userSchema.index({ schoolId: 1, roles: -1 });
+userSchema.index({ firstName: 'text', lastName: 'text', email: 'text', searchIndexes: 'text' });
```

Comment:
Is it still necessary to index the fields themselves with the searchIndexes attribute?